README.md

```diff
@@ -56,6 +56,21 @@ output a raw BIOS bootable disk image, or `linuxkit build -format iso-efi linuxk
 
 Since `linuxkit build` is built around the [Moby tool](https://github.com/moby/tool) the input yml files are described in the [Moby tool documentation](https://github.com/moby/tool/blob/master/docs/yaml.md).
 
+
+### Baking images
+```
+linuxkit bake linuxkit_template.yml
+```
+This will replace all angle bracketed tags in the image field by its hash defined in `build.yml`. If the tag is `<latest>` it gets replaced by the current pkg directorys git subtree hash. The location of the pkg directory is either defined by parameter `--pkgroot org:path` or in the global config, e.g.:
+
+```
+repos:
+  - path: /go/src/github.com/linuxkit/linuxkit/pkg
+    org: linuxkit
+  - path: /go/src/github.com/kubernetes/kubernetes/pkg
+    org: linuxkit
```

Comment:
Please define the meaning of the fields. I guess that `path` is searched for subdirectories (nested? or one level?) which might be packages (based on them containing a `build.yml`?).

I don't understand the need for `org` here at all, and packages under `path` will have an org as part of their `build.yml` (or will use the default). Perhaps it is an optimisation to avoid searching that path if the org doesn't match? That seems like a premature optimisation to me, but if this field is to exist (and I've inferred the reason correctly) it should accept the possibility that a given repo might have packages targetting multiple orgs, and hence be `orgs` and an array.

Reply:
@rn suggested it. I dont think performance is relevant soon so dont care to much. I can remove, leave it, or make it an array.

updated the docs.

Reply:
yes, I did suggest the `org` part, but that was because I forgot that `build.yml` contains the information. My initial intention was not to use `build.yml` but it does make sense since people may have directory name != package name.

Reply:
lets remove it for now.

Reply:
done.